### DISCUSSION TITLE
Homeassistant: All sensors: unknown

### DISCUSSION THREAD

**Discussion by FuxPCN** [raomin/ESPAltherma#405] [General]
Added template-section in HA config following the example, but all sensors are shown as unknown.

Where do I find the names for the sensors? In the example https://raomin.github.io/ESPAltherma/ there is an attribute "I/U operation mode" at the screenshot. And in the template the name it is "Indoor Operation mode" and the unique name is "iuoperation".

Where is the mapping between these three different names? And how do I get the mappings for all other sensors?

I took the names from the attribute list in HA unter "AlthermaSensors" but it doesn't work.

**Comment by redtree99** [comment]
There is a good guide here: https://community.openenergymonitor.org/t/daikin-altherma-espaltherma-home-assistant-with-openenergymonitor/25057/6

For some reason my entity was different ('sensor.none_althermasensors') so double check yours is correct. You can do this by clicking on the search bar at the top and looking for it.  See my pic:

![image](https://github.com/raomin/ESPAltherma/assets/162464600/fe72da66-7480-4092-82a9-145593aa64a5)
